### PR TITLE
Change structure in DropDownMenu

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**
I used tag's ul li in HTML structure instead of select in HEADER-Drop down menu ("Select a category") and also added some styles for Drop down menu !!

### CODE REVIEW

File: src/sass/components/_header.scss [hunk 1]
```diff
@@ -168,32 +168,60 @@ header {
...
+          left: -1px;
+        }
+
+        &:hover .select-category-list {
+          display: block;
+          visibility: visible;
+          cursor: pointer;
+        }
+
+        li {
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: please use classes for styling

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -168,32 +168,60 @@ header {
...
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          padding: 12px;
+          list-style: none;
+
+          &:hover {
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: use @mixin for hover

File: src/sass/components/_header.scss [hunk 3]
```diff
@@ -168,32 +168,60 @@ header {
...
+            background-color: $primary;
+            color: #fff;
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: color from variables

File: src/sass/components/_header.scss [hunk 4]
```diff
@@ -168,32 +168,60 @@ header {
...
-          z-index: 1;
+          list-style: none;
+          margin: 0;
+        }
+
+        .select-category-list {
+          width: 101%;
+          visibility: collapse;
+          position: absolute;
+          background-color: #fff;
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: color from variables

File: src/sass/components/_header.scss [hunk 5]
```diff
@@ -168,32 +168,60 @@ header {
...
+          padding: 0 40px;
           font-size: 14px;
-          position: relative;
-          z-index: 1;
+          list-style: none;
+          margin: 0;
+        }
+
+        .select-category-list {
+          width: 101%;
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: why 101%?

File: src/sass/components/_header.scss (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: class added !!

File: src/sass/components/_header.scss (hunk 2 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: I created file mixins.scss in folder mixins, and added @mixin for hover

File: src/sass/components/_header.scss (hunk 3 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: done !!

File: src/sass/components/_header.scss (hunk 4 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: done !!

File: src/sass/components/_header.scss (hunk 5 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: I changed that !!

File: src/sass/components/_header.scss [hunk 6]
```diff
@@ -212,17 +216,13 @@ header {
...
           justify-content: center;
           align-items: center;
           padding: 12px;
           list-style: none;
-
-          &:hover {
-            background-color: $primary;
-            color: #fff;
-          }
+          @include hover-drop-down-menu;
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: this hover is still working on devices that doesn't supports hover

File: src/sass/mixins/_mixins.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+@mixin hover-drop-down-menu {
```
Comment (matwjablonski) [inline src/sass/mixins/_mixins.scss]: this file is unnecessary

File: src/sass/style.scss [hunk 8]
```diff
@@ -13,6 +13,7 @@
 // MIXINS
 @import "mixins/fade";
 @import "mixins/hover-supported-devices";
+@import "mixins/_mixins.scss";
```
Comment (matwjablonski) [inline src/sass/style.scss]: imports without `_`

File: src/sass/components/_header.scss (hunk 6 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_header.scss]: @mixin hover-only added !!

File: src/sass/mixins/_mixins.scss (hunk 7 above)
Comment (MiroslawChmielecki) [reply src/sass/mixins/_mixins.scss]: deleted !!

File: src/sass/style.scss (hunk 8 above)
Comment (MiroslawChmielecki) [reply src/sass/style.scss]: deleted !!

File: src/sass/_variables.scss [hunk 9]
```diff
@@ -1,5 +1,6 @@
 $header-topbar-bg: #864343;
 $header-bg: #723636;
+$header-box-shadow-color: rgba(1, 2, 2, 0.2);
```
Comment (thisisui) [inline src/sass/_variables.scss]: please change 0.2 into .2

Comment (MiroslawChmielecki) [reply src/sass/_variables.scss]: Chande 0.2 into .2 or into 2 ???
If I change 0.2 into .2 and commit that, display again 0.2 :/

Comment (matwjablonski) [reply src/sass/_variables.scss]: this is husky problem :) we need to reconfigure that tool ;)

**Pull request merged**